synapse/http/client.py

```diff
@@ -434,6 +436,10 @@ def get_json(self, uri, args={}, headers=None):
 
             ValueError: if the response was not JSON
         """
+        actual_headers = {b"Accept": [b"application/json"]}
```

Comment:
Hmm. I wonder whether this should also be setting the User-Agent.

Reply:
That was a discussion on #synapse-dev:matrix.org. I initially had those in the handler because some providers expected to have that header and I thought it would make sense to have them by default on all `*_json` methods in the http client.

Reply:
Oh, nevermind. Did not read your comment properly. That's actually done by `SimpleHttpClient.request`.